SlowCheetah.VisualStudio/NugetHandler/SlowCheetahNuGetManager.cs

```diff
@@ -39,6 +64,40 @@ public SlowCheetahNuGetManager(IServiceProvider package)
         }
 
         /// <summary>
+        /// Checks if the current project supports NuGet
+        /// </summary>
+        /// <param name="hierarchy">Hierarchy of the project to be verified</param>
+        /// <returns>True if the project supports NuGet</returns>
```

Comment:
Add a `<remarks>` tag saying that this implementation is borrowed from NuGet and link to their implementation. Also mention this should be removed when/if NuGet adds this to their public API.